documentation/developer-guide/modules/tooling-guide/pages/samm-cli.adoc

```diff
@@ -574,6 +586,339 @@ And finally for the time based paging:
+    "description" : "Aspect for movement information" // <3>
+  }
+}
+----
+
+<1> depending on parameters used when generating the specification, this is either the major version of the full Aspect URN (*2*.0.0), or it can be the full version (`v2.0.0`), if using `-sv` (semantic version) command line switch
+<2> as `samm:preferredName` is used as the value for the `title` element of the specification (MQTT API defined automatically)
+<3> as `samm:description` is an optional element
+
+The name of the Aspect is used to generate several important AsyncAPI artifacts, like the channel definitions for the API:
+
+[source,JSON]
+----
+{
+ "channels" : {
+    "Movement" : { // <3>
+      "address" : "/movement/0.0.1/Movement", // <3>
+      "description" : "This channel for updating Movement Aspect.", // <3>
+      "parameters" : {
+        "namespace" : "movement",
+        "version" : "0.0.1",
+        "aspect-name" : "Movement" // <3>
+      },
+      "massages" : {}
+    }
+  }
+}
+----
+
+==== Mapping of Aspect's operations
+
+AsyncAPI specification is generated using operations and events.
+This section describe only operations.
+If the Aspect also has a non-empty list of operations defined, like the one in the following example:
+
+[source,turtle]
+----
+:Movement a samm:Aspect ;
+   samm:preferredName "movement"@en ;
+   samm:description "Aspect for movement information"@en ;
+   samm:properties ( ) ;
+   samm:operations ( :GetSpeed ) ;
+   samm:events ( ) .
+
+:GetSpeed a samm:Operation ;
+   samm:preferredName "GetSpeedPrefName"@en ;
+   samm:description "GetSpeed summary"@en ;
+   samm:input ( :GetSpeedInput ) ; <1>
+   samm:output :GetSpeedOutput . <2>
+
+:GetSpeedOutput a samm:Property ;
+   samm:preferredName "GetSpeedOutputPrefferedName"@en ;
+   samm:description "GetSpeedOutput desc"@en ;
+   samm:characteristic :OutputCharacteristic .
+
+:OutputCharacteristic a samm:Characteristic ;
+   samm:preferredName "OutputCharacteristicPrefferedName"@en ;
+   samm:description "OutputCharacteristic desc"@en ;
+   samm:dataType :OutputEntity .
+
+:OutputEntity a samm:Entity ;
+   samm:name "GetSpeedOutput" ;
+   samm:preferredName "GetSpeedOutputPrefferedName"@en ;
+   samm:description "GetSpeedOutput desc"@en ;
+   samm:properties ( :OutputEntityMessage ) .
+
+:OutputEntityMessage a samm:Property ;
+   samm:characteristic samm-c:Text . <3>
+
+:GetSpeedInput a samm:Property ;
+   samm:name "getSpeedInput" ;
+   samm:preferredName "getSpeedInputPrefferedName"@en ;
+   samm:description "getSpeedInput desc"@en ;
+   samm:characteristic samm-c:Text . <4>
+----
+
+operations will be generated in some places of AsyncAPI specification:
+https://www.asyncapi.com/docs/reference/specification/v3.0.0#operationObject[operations] section,
+channel
+https://www.asyncapi.com/docs/reference/specification/v3.0.0#messagesObject[messages] section and
+https://www.asyncapi.com/docs/reference/specification/v3.0.0#schemaObject[schemas] section:
+
+[source,JSON]
```

Comment:
In the following AsyncAPI schema document, change identifiers according to the Aspect Model changes (Property and Operation names lowercase, updates to preferredNames and descriptions)